projects/ReOxide/services/reoxided/module.nix

```
@@ -0,0 +1,73 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkEnableOption
+    mkPackageOption
+    mkOption
+    mkIf
+    types
+    ;
+
+  cfg = config.services.reoxided;
+  toToml = pkgs.formats.toml { };
```

Just a nit, but generally you'd see this called `settingsFormat` in Nixpkgs.


```suggestion
  settingsFormat = pkgs.formats.toml { };
```